backend/preprocess/ChannelCache.py

```diff
@@ -16,7 +16,7 @@ def __init__(self):
 		if is_compiled():
 			self.cache_folder_path = "../../storage/cache/message-ids"
 		else:
-			self.cache_folder_path = "../../release/dcef/storage/cache/message-ids"
+			self.cache_folder_path = "../../cache/message-ids"
```

Comment:
This change will break development environment. I will restructure the project folder structure to fix paths later

Reply:
Is there anything I can do to resolve this to make it easier on you? I am on my way to bed now as its 11PM here, but after I get back from school tomorrow I can make the changes. Where would I look to find the breakage?